patches/minecraft/net/minecraft/entity/EntityType.java.patch

```diff
@@ -95,19 +94,18 @@
     public static class Builder<T extends Entity> {
        private final EntityType.IFactory<T> field_200709_a;
        private final EntityClassification field_220324_b;
-@@ -531,6 +567,11 @@
+@@ -530,6 +565,10 @@
+       private int field_233604_h_ = 5;
        private int field_233605_i_ = 3;
        private EntitySize field_220326_f = EntitySize.func_220314_b(0.6F, 1.8F);
- 
 +      private java.util.function.Predicate<EntityType<?>> velocityUpdateSupplier = EntityType::defaultVelocitySupplier;
 +      private java.util.function.ToIntFunction<EntityType<?>> trackingRangeSupplier = EntityType::defaultTrackingRangeSupplier;
 +      private java.util.function.ToIntFunction<EntityType<?>> updateIntervalSupplier = EntityType::defaultUpdateIntervalSupplier;
 +      private java.util.function.BiFunction<net.minecraftforge.fml.network.FMLPlayMessages.SpawnEntity, World, T> customClientFactory;
-+
+ 
```

Comment:
And us move a lot of these Patches to Mixins, and thus allow easier updating and verifications.